.github/workflows/nightly-wheels.yml

```
@@ -0,0 +1,41 @@
+name: Upload nightly wheels
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
+  cron:
+    runs-on: ubuntu-latest
```

We don't want to run this on any fork:
```suggestion
    runs-on: ubuntu-latest
    if: github.repository == 'pydata/xarray'
```